### PR TITLE
Notifications 2.0

### PR DESCRIPTION
**Pull request by ShadowSpin1**

### CODE REVIEW

File: app/controllers/guests_controller.rb [hunk 1]
```diff
@@ -64,16 +67,45 @@ def set_guest
 
   def notify_host
     host = @guest.event.host
-    notification = GuestNotification.with(type: "join", host: host,
-                                          recipient: @guest.guest, status: @guest.status, event: @guest.event)
-    pp notification
+    notification = GuestNotification.with(
+      type: "event",
+      message: "#{@guest.guest.first_name} would like to join your event.",
+      recipient: @guest.guest,
```
Comment (laura-gonzalez-z) [inline app/controllers/guests_controller.rb:73]: Just to clarify... recipient here shouldn't be the host? I know that the important part is in line 76 but would like to understand what recipient does here.

File: app/controllers/guests_controller.rb [hunk 2]
```diff
@@ -64,16 +67,45 @@ def set_guest
...
+    )
     notification.deliver(guest)
   end
+
+  def notify_guest_accept
+    guest = @guest.guest
+    notification = GuestNotification.with(
+      type: "event",
+      message: "#{@guest.event.host.first_name} accepted your request to join this event.",
+      recipient: @guest.guest,
```
Comment (laura-gonzalez-z) [inline app/controllers/guests_controller.rb:95]: Can they be replaced by `recipient: guest` when it's the same as `@guest.guest`?

**Review by laura-gonzalez-z — COMMENTED** [review]
The rest look good to me. Thanks for refactoring the footer, it's great!

File: app/controllers/guests_controller.rb (hunk 1 above)
Comment (ShadowSpin1) [reply app/controllers/guests_controller.rb:73]: recipient of the notification would be the host, but recipient in the params is the guest!

File: app/controllers/guests_controller.rb (hunk 2 above)
Comment (ShadowSpin1) [reply app/controllers/guests_controller.rb:95]: hmmm could be but I kept it the same across everything.

**Review by laura-gonzalez-z — APPROVED** [review]

**Pull request merged**